src/index.js

```diff
@@ -0,0 +1,34 @@
+import './style.css';
+
+const tasks = [{
+  description: 'Coding milestone projects',
+  completed: false,
+  index: 1,
+},
+{
+  description: 'Learning JS ES6',
+  completed: false,
+  index: 2,
+},
+{
+  description: 'Attending morning session meeting',
+  completed: false,
+  index: 3,
+},
+{
+  description: 'Going for lunch break',
+  completed: false,
+  index: 4,
+},
+];
+
+const itemList = document.getElementById('addItem');
+
+function item() {
+  itemList.innerHTML = tasks.map((task) => `<li class="items">
+    <label for="items">${task.description}</label>
+</li>`).join();
+}
+window.onload = () => {
+  item();
+};
```

Comment:
- [ ] While you did a wonderful job adding the required layout to your project :+1:, as a [project requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m1_list_structure.md), your list should be a clone of the part of the minimalist project captured in the [video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). It seems like you miss some elements like the checkbox and the required icons. Kindly add those elements to match as much as possible with the design

**Your design**

![Screenshot from 2023-02-22 21-17-37](https://user-images.githubusercontent.com/105117832/220807312-cef131a1-20ab-4a0b-a30b-ca7f522fdbbf.png)


**Figma**

![Screenshot from 2023-02-22 21-12-52](https://user-images.githubusercontent.com/105117832/220807244-ef6aeb54-a1da-47e7-8375-f22eadab7538.png)